apiary.apib

```diff
@@ -3360,6 +3362,17 @@ Update a milestone.
 
 + Response 204 (application/json)
 
+### milestones.delete [POST /milestones.delete]
+
+Delete a milestone.
```

Comment:
You are not able to delete the last milestone right? Can we add some more information here to help out the developers using the API? :)

Reply:
The earlier we can make it clear in the documentation the better though ;)